bin/improver-weighted-blending

```diff
@@ -197,6 +199,22 @@ def main():
     parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
                         help='The output path for the processed NetCDF.')
 
+    spatial = parser.add_argument_group(
+        'Spatial weights from mask options',
+        'Options for calculating the spatial weights using the '
+        'SpatiallyVaryingWeightsFromMask plugin.')
+    spatial.add_argument('--fuzzy_length', metavar='FUZZY_LENGTH', type=float,
+                         default=10,
```

Comment:
I would prefer to see this as a distance in metres or kilometres please, for consistency with other CLIs.

Reply:
I can change this, but I think it would need a plugin change to convert the distance to a number of grid points.

Reply:
You could call the spatial utilities function from the CLI or, yes, from the plugin if needed.

Did we have this conversation about the plugin already?

Reply:
Yes we did. See this discussion: https://github.com/metoppv/improver/pull/753#discussion_r238225965

Reply:
Ah I see.  I think I'm still OK with that decision at the plugin level, but it doesn't sit right with me for a CLI to be taking arguments in subjective units.  Perhaps you could do a float conversion in the CLI?

Reply:
I've now done this by adding the option to return a float for the number of grid squares to the function we already had to convert from m to grid lengths. I've then added a call to this function to the CLI.